docs/root/api/starting_envoy.rst

```diff
@@ -163,6 +163,27 @@ This information is sent as metadata when flushing stats.
   // Swift
   builder.addAppId("com.mydomain.myapp)
 
+~~~~~~~~~~~~~~~~~~~~~~
+``addVirtualClusters``
+~~~~~~~~~~~~~~~~~~~~~~
+
+Specify the virtual clusters config for Envoy Mobile's configuration.
+The configuration is expected as a JSON list.
+This functionality is used for stat segmentation.
+
+.. attention::
+
+    This API is non-ideal as it exposes lower-level internals of Envoy than desired by this project.
+    https://github.com/lyft/envoy-mobile/issues/770 tracks enhancing this API.
+
+**Example**::
+
+  // Kotlin
+  builder.addVirtualClusters("[{json_config}]")
```

Comment:
Is there an example if a `json_config` or is it just a list of strings?

Reply:
Definitely agree that we should add an actual example here